zio-http/src/main/scala/zhttp/service/Server.scala

```diff
@@ -69,6 +77,22 @@ object Server {
     (Server.port(port) ++ Server.app(http)).make.useForever
       .provideSomeLayer[R](EventLoopGroup.auto(0) ++ ServerChannelFactory.auto)
 
+  def start[R <: Has[_]](
+    hostname: String,
+    port: Int,
+    http: RHttpApp[R],
+  ): ZIO[R, Throwable, Nothing] =
+    (Server.port(port) ++ Server.app(http) ++ Server.Hostname(hostname)).make.useForever
+      .provideSomeLayer[R](EventLoopGroup.auto(0) ++ ServerChannelFactory.auto)
+
+  def start[R <: Has[_]](
+    hostAddress: InetAddress,
+    port: Int,
```

Comment:
```suggestion
```

Port isn't required with `hostAddress`

Reply:
Port will be required for InetAddress. It will not be required for InetSocketAddress,  should I add an overload for InetSocketAddress?

Reply:
Right, Another overload for SocketAddress in that case :)